apps/backend/convex/entities/types.ts

```diff
@@ -22,6 +22,7 @@ export type ThreadWithDetails = Infer<typeof ThreadWithDetailsReturn>
 export type User = Infer<typeof UserReturn>
 
 export type MessageRoles = Infer<typeof MessageRoles>
+export type ModelParameters = Infer<AsObjectValidator<typeof ModelParameters>>
```

Comment:
It is a common pattern to define a validator and the inferred type of the validated value with the same name. It may be confusing to you due to your limited context, but not to a human who is looking at the codebase. There is no type conflict as the imported values are not types. Please refrain from making similar suggestions on inferred validator types in the future.